R/clustering.R

```diff
@@ -155,7 +152,6 @@ FindClusters.Seurat <- function(
   graph.name = NULL,
   modularity.fxn = 1,
   initial.membership = NULL,
-  weights = NULL,
```

Comment:
```suggestion
  weights = NULL,
```

Keep for next release and remove in following one.